lib/core/src/server/config/utils.js

```diff
@@ -2,38 +2,37 @@ import path from 'path';
 import { getEnvironment } from 'lazy-universal-dotenv';
 
 export const includePaths = [path.resolve('./')];
-
-export const excludePaths = [path.resolve('node_modules')];
-
 export const nodeModulesPaths = path.resolve('./node_modules');
-
-export const nodePaths = (process.env.NODE_PATH || '')
-  .split(process.platform === 'win32' ? ';' : ':')
-  .filter(Boolean)
-  .map(p => path.resolve('./', p));
+export const excludePaths = [nodeModulesPaths];
```

Comment:
Can we revert this changes in favor of #4706 ?

Reply:
I'm unsure exactly what it'd be that I'd have to change here.